### PR TITLE
(improvements) Weighted Averages report updates

### PR DESCRIPTION
**Pull request by alexstotsky**
Task: https://app.asana.com/0/1163495710802945/1204955530705536/f

#### Description:
- [x] Reworks `Weighted Averages` report and corresponding handling logic according to the latest requirements:
- Only one pair should be selected and shown at a time, `BTC/USD` set by default
- Set the default [date range](https://bitfinex.slack.com/archives/GAVNZP4RF/p1688477351665119?thread_ts=1688219520.121779&cid=GAVNZP4RF) to `Past Month`
- Limit notice changed to the following:
```
Not all transactions could be displayed
Your request exceeds the limit of 50k transactions.
Download the Reports App to display more or change your date range.
```

#### Preview:
weighted_avgs_single _pair

#### Depends on: 
- [bfx-reports-framework #292](https://github.com/bitfinexcom/bfx-reports-framework/pull/292)
- [bfx-report #306](https://github.com/bitfinexcom/bfx-report/pull/306)

### CODE REVIEW

File: public/locales/en/translations.json [hunk 1]
```diff
@@ -723,10 +723,10 @@
   "weightedaverages": {
     "title": "Weighted Averages",
     "limitNote":{
-      "displayDate": "Displaying transactions for period: ",
-      "note1": "Your request exceeds the limit of 2000 transactions to display.",
+      "title": "Not all transactions could be displayed",
+      "note1": "Your request exceeds the limit of 100k transactions.",
```
Comment (ezewer) [inline public/locales/en/translations.json:727]: Small comment here, should be 50K

**Review by ezewer — APPROVED** [review]
Lgtm, small comment!

File: public/locales/en/translations.json (hunk 1 above)
Comment (alexstotsky) [reply public/locales/en/translations.json]: Updated

**Review by ezewer — APPROVED** [review]
lgtm!

**Pull request merged**